lib/screens/AuthenticationScreen.dart

```diff
@@ -0,0 +1,229 @@
+                  fontSize: 20,
+                  color: COLOR_BLUE_WHITE),
+              hintText: hint,
+              focusedBorder: OutlineInputBorder(
+                  borderSide: BorderSide(color: COLOR_BLUE_WHITE, width: 3)),
+              enabledBorder: OutlineInputBorder(
+                  borderSide: BorderSide(color: COLOR_WHITE, width: 1)),
+              prefixIcon: Padding(
+                padding: EdgeInsets.only(left: 10, right: 10),
+                child: IconTheme(
+                  data: IconThemeData(color: COLOR_WATER),
+                  child: icon,
+                ),
+              )),
+        ),
+      );
+    }
+
+    Widget _button(String text, void foo()) {
+      return ElevatedButton(
+        onPressed: foo,
+        style: ElevatedButton.styleFrom(
+          primary: COLOR_BLUE_WHITE,
+          shadowColor: COLOR_BLACK,
+          onPrimary: COLOR_BUTTON,
+        ),
+        child: Text(
+          text,
+          style: TextStyle(
+              fontWeight: FontWeight.bold,
+              color: COLOR_DARK_BLUE,
+              fontSize: 20),
+        ),
+      );
+    }
+
+    Widget _form(String label, void foo()) {
+      return Container(
+        child: Column(
+          children: [
+            Padding(
+              padding: EdgeInsets.only(bottom: 20, top: 70),
+              child:
+                  _input(Icon(Icons.email), 'EMAIL', _emailController, false),
+            ),
+            Padding(
+              padding: EdgeInsets.only(bottom: 20, top: 0),
+              child: _input(
+                  Icon(Icons.lock), 'PASSWORD', _passwordController, true),
+            ),
+            SizedBox(
+              height: 20,
+            ),
+            Padding(
+              padding: EdgeInsets.only(left: 20, right: 20, bottom: 5),
+              child: Container(
+                height: 50,
+                width: MediaQuery.of(context).size.width,
+                child: _button(label, foo),
+              ),
+            )
+          ],
+        ),
+      );
+    }
+
+    void showToast(String text) {
+      Fluttertoast.showToast(
+          msg: text,
+          toastLength: Toast.LENGTH_SHORT,
+          gravity: ToastGravity.CENTER,
+          backgroundColor: Colors.red,
+          textColor: Colors.white,
+          fontSize: 50);
+    }
+
+    void _loginUser() async {
+      _email = _emailController.text.trim();
+      _password = _passwordController.text.trim();
+
+      if (!EmailValidator.validate(_email)) {
+        showToast("Please check email\nThen repeat again");
+        return;
+      }
+      if (!passwordValidator.validate(_password)) {
+        showToast("Please check password\nThen repeat again");
+        return;
+      }
+
+      MyUser.User user =
+          await _authService.signInWithEmailAndPassword(_email, _password);
+      if (user == null) {
+        showToast("Please check email and password\nThen repeat again");
```

Comment:
We need to use string resources for better interactions and to give an opportunity to translate our app